nativelink-store/tests/redis_store_test.rs

```diff
@@ -980,3 +847,23 @@ async fn dont_loop_forever_on_empty() -> Result<(), Error> {
 
     Ok(())
 }
+
+#[nativelink_test]
+fn test_connection_errors() {
+    let spec = RedisSpec {
+        addresses: vec!["redis://non-existent-server:6379/".to_string()],
+        command_timeout_ms: 10,
+        ..Default::default()
+    };
+    let store = RedisStore::new(spec).expect("Working spec");
+    let err = store
+        .has("1234")
+        .await
+        .expect_err("Wanted connection error");
+    assert_eq!(err.messages.len(), 2);
+    // err.messages[0] varies a bit, always something about lookup failures
```

Comment:
varies a bit by OS, probably? I know we have people running on Mac runners and Windows runners (somehow).

Reply:
ahh, so by a network and other factors.